scipy/sparse/tests/test_base.py

```diff
@@ -4666,17 +4667,23 @@ def check(cls, method_name):
 
         check(cls, method_name)
 
+    @pytest.mark.xfail(platform.machine() == 'ppc64le',
+		       reason="fails on ppc64le")
```

Comment:
tests in this module were highly problematic on the GCC compile farm ppc64le machine, and also apparently not failing reliably, so that `xpass` counts get inflated.

Reply:
Not sure if CentOS / default config there is causing additional issues beyond those we may observe in the Travis Ubuntu run on ppc64le.